src/settings.cc

```diff
@@ -127,6 +127,7 @@ Napi::Value getTraceSettings(const Napi::CallbackInfo& info) {
       if (xtrace.length() == 60) {
         // try to convert it to metadata. if it fails act as if no xtrace was
         // supplied.
+        oboe_metadata_init(&omd);
```

Comment:
you found the needle in the hay stack ;)

Reply:
crawling under hay stack needle found me ;)